### ISSUE TITLE
[Clock App] Alarm silenced when phone is in silent mode

### ISSUE THREAD

**Issue by Th0masL** [GrapheneOS/os-issue-tracker#3249]
Hey guys !

Thanks for this great OS !

I've been using GrapheneOS since 2 weeks on my Pixel 8, and I realized today that if I put my phone in silence mode, the alarms are also silenced.

I am surprised by this behavior, because I'm pretty sure it was not happening with my previous phone, a Galaxy S10e, that was running LineageOS 20 (Android 13).

After some investigation, it looks like the Clock app in GrapheneOS is using sounds as "ringtones", and not as "notifications". This also explains why I can't select any of the sounds that are in the "notifications" category as Alarm sounds, and it's only offering the choice in the "ringtones" category.

Basically it seems like the Clock app falls in the same category as the Phone app, and that's why it's getting silenced when the phone is set in silence mode.

On all the android phones I used in the past, the Clock app was never impacted by the phone's sound settings, and was always ringing correctly, even if the phone was in silent.

I'm adding 2 screenshots, to show the difference between my old Galaxy S10e on LineageOS20 and my Pixel 8 on the latest GrapheneOS.

Samsung Galaxy S10e on LineageOS20 (Android 13)
![s10e](https://github.com/GrapheneOS/os-issue-tracker/assets/25387544/2ad687e2-34f3-4868-aea2-cc6e1f362ce2)

Pixel8 on GrapheneOS (Android 14):
![pixel8](https://github.com/GrapheneOS/os-issue-tracker/assets/25387544/e69b3576-f51a-4eff-ae0d-49c2cdc363bd)

It might be an expected behavior from the Clock app, but this behavior is different than all the phones I had in the past, and it's annoying to have to remember to remove the phone from silent mode at night, as I don't want to be woken up by phone calls, but I still want the alarm to wake me up.

Let me know if you need more information.

Thanks

**Comment by Th0masL** [comment]
I just tried with another clock (Simple Clock, from F-Droid [here](https://f-droid.org/en/packages/com.simplemobiletools.clock/)).

This clock app is offering to chose the sound in the "Alarm" category, which looks better than "Ringtones".

![pixel8-simple-clock](https://github.com/GrapheneOS/os-issue-tracker/assets/25387544/aec6c449-2585-4734-84a6-8a4c8198b381)


But the sound is still being silenced when the phone is in silent mode.

This could be an Android 14 feature, or it could be something that only impact GrapheneOS.

I will try to get another phone on Android 14 to compare the behavior.

**Comment by Th0masL** [comment]
Thanks for asking for clarification @matchboxbananasynergy.

The fact that you confirmed it was working for you made me verify everything again on my side.

I realized I had set my alarm to "Silent" in the Clock app itself, couple of days ago, as I needed a special non-ringing alarm, and I forgot to re-enable the sound. This is why my alarm would never make noise.

When I was trying to troubleshoot, I went in the app's global settings and I was seeing that I could only select a sound from the "Ringtones" list, so that got me thinking that the Clock app was being applied the same settings as the Phone app, and that's why it would be silenced when the phone was in silent mode.

So yeah, long story short, the Clock app is ringing correctly even in silent mode. It's possible that the Clock app is not tagged as a proper "Alarm" type app, that's why we can't see the list of alarm sounds in the selection menu of the app itself, but it probably doesn't matter, as the sound can be selected on a per-alarm basis, on every alarm.

I made a new screenshot to explain why I got confused.

![pixel8-alarm-config](https://github.com/GrapheneOS/os-issue-tracker/assets/25387544/5a91c668-8820-44f2-9483-e4ed46e9c7ec)

I will close this issue, as I don't think it's needed to change the Clock app's default tagging as "Alarm" simply to show the list of Alarm sounds.
